debian/rules

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/make -f
+
+VERSION=0.1
+DEB_DIR=debian/streamgraph$(VERSION)
+
+%:
+	dh $@
+
+override_dh_auto_clean:
```

Comment:
What breaks from clean?  Maybe this just needs to call `cmake` to generate the `Makefile` in order to do a `make clean`?